CMakeLists.txt

```diff
@@ -74,17 +75,20 @@ foreach (SRC_FILE IN LISTS CPRNC_GenF90_SRCS)
     DEPENDS ${CMAKE_CURRENT_SOURCE_DIR}/${SRC_FILE}.in genf90)
 endforeach ()
 
+#===== NetCDF =====
+pkg_check_modules(NetCDF REQUIRED IMPORTED_TARGET netcdf)
```

Comment:
Does this work with static libraries?

https://github.com/NOAA-EMC/CMakeModules/blob/9b7652c75b40d9cbb40e52b824f8c0a423922757/Modules/FindNetCDF.cmake#L232